### PR TITLE
Made disco_ssh.py able to be decisive.

### PR DESCRIPTION
**Pull request by bwarfield-amplify**
Added an option to disco_ssh.py to allow connecting to the first instance in the matching list, rather than requiring an unambiguous instance selector.

### CODE REVIEW

**Comment by aoskotsky-amplify** [comment]
💯

File: bin/disco_ssh.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
      -h --help              Show this screen
      --debug                Log in debug level
      --env ENV              Environment to operate in
+     --first                In case of multiple matching instances, connect to the first instead of failing
```
Comment (eugenejen) [inline bin/disco_ssh.py]: how about --any? there is really no ordering in picking right?
(maybe you will have --earliest, --latest)

Comment (bwarfield-amplify) [inline bin/disco_ssh.py]: I don't really care one way or the other—picking `instances[0]` was easier than randomizing, and I didn't see any reason to pretend that I was randomizing internally.

**Comment by pchand-amplify** [comment]
What happens when you run 'disco_SSH.py webapp'?

**Comment by bwarfield-amplify** [comment]
Then you get a message saying you tried to connect to one of like twenty
hosts from 10 hostclasses, and you're getting this one, and you probably
reconsider your life choices. :-) That's one reason I wouldn't want to make
it the default, though.

**Comment by pchand-amplify** [comment]
So if I run `disco_ssh.py webapp --first`, will it ssh into the first instance of mhcwebapp or is it going to be any one of the hostclasses that has the word "webapp" in it?

**Comment by bwarfield-amplify** [comment]
The latter.  If you use "mhcwebapp" you will of course get what you expected.

**Comment by pchand-amplify** [comment]
Then its going to mislead people into believing that they are in the right hostclass. If we can detect instances from multiple hostclass and throw an error, then 👍  for `--first` else 👎 .

**Comment by bwarfield-amplify** [comment]
That seems unnecessarily solicitous to me, but it's totally easy to do, so
sure. :-)

**Comment by bwarfield-amplify** [comment]
Probably merits a brief re-review, since I changed the implementation significantly to address @pchand-amplify's feedback.

**Comment by pchand-amplify** [comment]
👍

**Comment by IlyaSukhanov** [comment]
+1

**Pull request merged**